jobbergate-composed/etc/jobbergate-local.json

```diff
@@ -87,6 +87,22 @@
     ],
     "client": {
       "cli": [
+        {
+          "id": "2dd32416-2e52-4ed3-97cd-b941753b7538",
```

Comment:
Weren't these changes to the jobbergate-local config for keycloak part of another PR?

Reply:
The cluster status has been implemented on PRs #538 and #539, both merged. The extra permissions implemented on them were missing here on local keycloak, so I'm fixing the issue.